apps/dsb-message-broker/src/auth/login.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { LoginReturnDataDTO } from './login-return-data.dto';
+import { ethers } from 'ethers';
+
+@Injectable()
+export class LoginService {
+    constructor(private readonly configService: ConfigService) {}
+
+    public async mbIdentifiers(jwtToken?: string): Promise<Partial<LoginReturnDataDTO>> {
```

Comment:
Why using optional ? here? Code will fail on the first line anyways jwtToken.split

Reply:
changed to required